src/main/java/com/exasol/sql/dql/select/FromClause.java

```diff
@@ -179,5 +203,6 @@ public void accept(final SelectVisitor visitor) {
         for (final ValueTable valueTable : this.valueTables) {
```

Comment:
Combining a sub-Select and a value table is illegal, so it should end in an exception. Also add a test that it does.

Reply:
done